fix for last open items at https://github.com/OWASP/owasp-masvs/issues/203 (#1479)
OWASP/owasp-mstg: Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

Internal requirements count=1 size=176
```

> There are various ways to distribute your app as described at [the Apple documentation](https://developer.apple.com/business/distribute/ "Apple Business") using the App Store, Custom or In-house distribution methodologies. In case of an In-house distribution scheme, make sure that no Ad-Hoc certificates are used when the app is signed for distribution.

Reviewer: ```suggestion
There are various ways to distribute your app as described in [the Apple documentation](https://developer.apple.com/business/distribute/ "Apple Business") using the App Store, custom or in-house distribution mechanisms. In case of an In-house distribution scheme, make sure that no Ad-Hoc certificates are used when the app is signed for distribution.
```

Reviewer 2: Second In-house also to lowercase :)

> There are various ways to distribute your app as described at [the Apple documentation](https://developer.apple.com/business/distribute/ "Apple Business") using the App Store, custom or in-house distribution methodologies. In case of an in-house distribution scheme, make sure that no Ad-Hoc certificates are used when the app is signed for distribution.

Reviewer 2: - corrected "ad hoc" that's written without hyphen. Could you please correct other occurrences as well?
- added little enhancement about Apple Business Manager

```suggestion
There are various ways to distribute your app as described at [the Apple documentation](https://developer.apple.com/business/distribute/ "Apple Business"), which include using the App Store or via Apple Business Manager for custom or in-house distribution. In case of an in-house distribution scheme, make sure that no ad hoc certificates are used when the app is signed for distribution.
```

- Libraries that are not (or should not) be packed within the actual production application, such as `OHHTTPStubs` used for testing.
- Libraries that are packed within the actual production application, such as `Alamofire`.

These libraries can lead to unwanted side-effects:

- A library can contain a vulnerability, which will make the application vulnerable. A good example is `AFNetworking` version 2.5.1, which contained a bug that disabled certificate validation. This vulnerability would allow attackers to execute man-in-the-middle attacks against apps that are using the library to connect to their APIs.

> - A library can no longer be maintained or hardly be used, which is why no vulnerabilities are reported and/or fixed. This can lead to having bad and/or vulnerable code in your application through the library.

Reviewer 2: Slightly re-phrased. What do you think? 

```suggestion
- A library is no longer maintained or is not widely used, which is why no vulnerabilities are reported and/or fixed. This can lead to having bad and/or vulnerable code in your application because of that library.
```

> These libraries can have the following two classes of unwanted side-effects:

Reviewer: ```suggestion
These libraries can lead to unwanted side-effects:
```

In the case of copy-pasted sources: search the header files (in case of using Objective-C) and otherwise the Swift files for known method names for known libraries.

> Next, please note that for hybrid applications, one will have to check the JavaScript dependencies with RetireJS. Similarly for Xamarin, one will have to check the C# dependencies.

Reviewer: ```suggestion
Next, note that for hybrid applications, you will have to check the JavaScript dependencies with RetireJS. Similarly for Xamarin, you will have to check the C# dependencies.
```
(Conforming to style guide)

> Last, if the application is a high risk application, you will end up vetting the library manually. In that case there are specific requirements for native code, which are similar to the requirements established by the MASVS for the application as a whole. Next to that, it is good to vet whether all best practices for software engineering are applied.

Reviewer 2: ```suggestion
Last, if the application is a high-risk application, you will end up vetting the library manually. In that case, there are specific requirements for native code, which are similar to the requirements established by the MASVS for the application as a whole. Next to that, it is good to vet whether all best practices for software engineering are applied.
```

Author: Fixed

> Last, If the application is a high risk application, you will end up vetting the library manually. In that case there are a wild amount of review requirements, which are similar to the requirements established by the MASVS for the applicaiton as a whole. Next to that, it is good to vet whether all best practices for software engineering are applied.

Reviewer: ```suggestion
Last, if the application is a high risk application, you will end up vetting the library manually. In that case there are specific requirements for native code, which are similar to the requirements established by the MASVS for the application as a whole. Next to that, it is good to vet whether all best practices for software engineering are applied.
```

> ##### Detecting the licenses used by the libraries of the application

Reviewer 2: ```suggestion
##### Detecting the Licenses Used by the Libraries of the Application
```
